common/containers/Tabs/Swap/components/BitcoinQR.tsx

```diff
@@ -1,31 +1,28 @@
 import QRCode from 'qrcode.react';
 import React, { Component } from 'react';
+import { SwapInput } from 'reducers/swap/types';
 
 interface Props {
   paymentAddress: string | null;
-  amount: number | null;
+  destination: SwapInput;
```

Comment:
should take amount, not destination object. This component will only need paymentAddress and the amount, it only uses destination to de-structure it to get `amount`.